### DISCUSSION TITLE
Emergency Fund - Cash & Asset

### DISCUSSION THREAD

**Discussion by Roxas72** [ghostfolio/ghostfolio#4121] [Q&A]
Hi, I'm in the middle of setting everything up, starting with a couple of ETFs and cryptos activities, and I've noticed that in the _Summary_ tab it auto-populated the Assets section of the E.F., as if it picked one the activity as beign an E.F. asset.
Is there any way to reconcile it? And what is the correct way to set the e.f.? Should I be adding an account with a cash balance and then writing that balance in the _Summary_? 
I hope I was clear enough in exposing my problem

**Comment by dtslvr** [comment]
Hi @Roxas72

When you create or edit an activity, you can set the tag _Emergency Fund_.
